src/tests/asset_test.ts

```diff
@@ -75,4 +76,29 @@ describe("Asset", () => {
       });
     });
   });
+
+  describe(".toAtomicAmount", () => {
```

Comment:
```suggestion
  describe("#toAtomicAmount", () => {
```

We use: `#` for instance method and `.` for static methods (this is at least a rubyism that has translated over here).